ozpricechecker/templates/pricefinderapp/userproduct_list.html

```diff
@@ -1,32 +1,32 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1  align="center">{{ user.get_username }}'s Product List</h1>
-
-    <!-- {% for user_product in userproduct_list %}
-    <ul>
-        <li><strong>User Product:</strong> {{ user_product.product }}</li>
-        <li><strong>Product price threshhold:</strong> {{ user_product.threshhold }}</li>
-    </ul>
-    {% empty %}
-        <li>No user product yet.</li>
-    {% endfor %}  -->
+    <h1 class="text-capitalize" align="center">{{ user.get_username }}'s Product List</h1>
 
     <table class="table">
         <tr>
             <thead class="thead-light">
                 <tr>
-                    <th scope="col">User Product</th>
-                    <th scope="col">Threshhold Price</th>
-                    <th scope="col">Action</th>
+                    <th scope="col">
+                        User Product
+                        <a class="badge badge-primary" href="{% url "userproduct_add" %}">+</a>
+                    </th>
+                    <th class="text-right" scope="col">Threshhold Price</th>
+                    <th class="text-right" scope="col">Price</th>
+                    <th scope="col">Curency</th>
+                    <th class="text-center" scope="col">DateTime</th>
```

Comment:
Can we name that a bit clearer to highlight the fact that it's the Product Price?

Reply:
Changed name to "Price Date"